api/egl.xml

```diff
@@ -149,6 +149,7 @@
     <enums namespace="EGLDRMBufferUseMESAMask" type="bitmask" comment="EGL_DRM_BUFFER_USE_MESA bits">
         <enum value="0x00000001" name="EGL_DRM_BUFFER_USE_SCANOUT_MESA"/>
         <enum value="0x00000002" name="EGL_DRM_BUFFER_USE_SHARE_MESA"/>
+        <enum value="0x00000004" name="EGL_DRM_BUFFER_USE_CURSOR_MESA"/>
```

Comment:
This needed to be added to the \<extension> block as well - I did this and regenerated the header after merging this.

Reply:
Indeed; should've grepped. Hopefully now I'll get it right in one go next time :)
Thanks for merging all my PR :+1: